packages/dma-contracts/contracts/libs/UseStorageSlot.sol

```diff
@@ -80,6 +71,8 @@ library StorageSlot {
     bytes32 param,
     uint256 paramMapping
   ) internal view returns (bytes32) {
+    uint256 length = _getLength(slotPosition);
+    require(paramMapping <= length, "StorageSlot: Index out of bounds");
     // if paramMapping is 0, return the param
     if (paramMapping == 0) return param;
```

Comment:
Just a suggestion, but I feel the `if (paramMapping == 0) return param;` is more likely to happen than the out of bounds case, so perhaps you can put it first to save some gas

Reply:
yes, you are right :) changed